packages/amplify-cli-pm/package.json

```diff
@@ -31,7 +30,7 @@
     "cli-kit": "^1.8.2",
     "listr": "^0.14.3",
     "npm-package-arg": "^8.0.1",
-    "ora": "5.0.0",
+    "ora": "5.1.0",
```

Comment:
This can probably be unpinned now, it's a leftover from when we used to pin it due to a bug

Reply:
Fixed